src/main/java/com/facebook/openwifirrm/ucentral/UCentralUtils.java

```diff
@@ -390,4 +390,20 @@ public static boolean isChannelInBand(int channel, String band) {
 		return LOWER_CHANNEL_LIMIT.get(band) <= channel
 				&& channel <= UPPER_CHANNEL_LIMIT.get(band);
 	}
+
+	/**
+	 * Given the channel, gets the band by checking lower bound and upper bound
+	 * of each band
+	 *
+	 * @param channel channel number
+	 * @return band if the channel can be mapped to a valid band; null otherwise
+	 */
+	public static String getBandFromChannel(int channel) {
```

Comment:
An alternate suggestion below, I'm not actually sure which is clearer/better practice. @elludraon thoughts?

`UCentralConstants.BANDS.stream().filter(b -> isChannelInBand(channel, b)).findAny().orElse(null);`

Reply:
This function should not exist (convo above)